integration-tests/plugins/__tests__/cart/store/carts.spec.ts

```diff
@@ -224,93 +285,6 @@ describe("Store Carts API", () => {
         })
       ).rejects.toThrow()
     })
-
-    it("should throw if sales channel is disabled", async () => {
```

Comment:
I moved these tests to `cart.workflows.spec.ts` to match against proper error messages.